### PR TITLE
UI components 2

### PR DESCRIPTION
**Pull request by Jokara998**

### CODE REVIEW

File: packages/ui-components/src/data-display/Card/Card.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      <Image
+        src={`${imageSrc ? imageSrc : ''}`}
+        alt={`${imageAlt ? imageAlt : "card-image"}`}
+      />
+    
+      <div className="p-4">
+        <h5 className="text-xl font-semibold mb-2">{title && title}</h5>
+    
+        <p className="mb-12">
+         {body && body}
```
Comment (lordsteva) [inline packages/ui-components/src/data-display/Card/Card.tsx]: i think you can just write {body}
react won't render falsy value

File: packages/ui-components/src/data-display/Card/Card.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React, { FC } from 'react';
+import { Image } from '../Image'
+import { Button } from '../../input-controls/Button'
+
+type Props = {
+  title?: string;
+  body?: string;
```
Comment (lordsteva) [inline packages/ui-components/src/data-display/Card/Card.tsx]: i'm not sure what this component will be used for, but think about making these (at least body) any, or jsx element
so we can provide custom body, not only string

**Review by lordsteva — APPROVED** [review]
good job young Jokaric 🥇 😄 🎉

File: packages/ui-components/src/data-display/Card/Card.tsx (hunk 2 above)
Comment (Jokara998) [reply packages/ui-components/src/data-display/Card/Card.tsx]: Nice suggestion :smile:

**Pull request merged**